modules/scheduler/gke-cluster/variables.tf

```diff
@@ -327,3 +327,15 @@ variable "additional_networks" {
     }))
   }))
 }
+
+variable "rdma_subnetwork_name_prefix" {
+  description = "Prefix of the RDMA subnetwork names"
+  default     = null
+  type        = string
+}
+
+variable "is_a3_ultra" {
```

Comment:
In the absence of the `machine_type` setting, needed a way to distinguish the a3 ultra in the `gke-cluster` module.

Reply:
Do you mean machine type for all the node_pools for the GKE cluster? For example, say a GKE cluster has the following nodepools:
* a3_ultra
* a3_mega
Would `is_a3_ultra` be true in that case? Also what happens when a3_ultra if customer removes a3_ultra nodepool but still passes this as `true` ?

Reply:
This is a temporary variable for the staging testing. You are right, even if that is that case, does not make sense to tie it to the machine type. I'll call it `is_staging`